assets/airports/kdtw.json

```diff
@@ -0,0 +1,4870 @@
+            },
+            "draw": [
+                ["_DXO296010", "DUNKS*"],
+                ["_DXO303010", "EARVN*"],
+                ["_DXO281010", "HARWL*"],
+                ["_DXO006010", "LAYNE*"],
+                ["_DXO239010", "RUZZL*"],
+                ["ANNTS*"]
+            ]
+        },
+        "RID8": {
+            "icao": "RID8",
+            "name": "Richmond Eight",
+            "rwy": {
+                "3L": [],
+                "3R": [],
+                "4L": [],
+                "4R": [],
+                "9L": [],
+                "9R": [],
+                "21L": [],
+                "21R": [],
+                "22L": [],
+                "22R": [],
+                "27L": [],
+                "27R": []
+            },
+            "body": [["_DXO202010", "A50+"], "CALVS", "VWV"],
+            "exitPoints": {
+                "RID": ["RID"]
+            },
+            "draw": [
+                ["_DXO202010", "CALVS", "VWV", "RID*"]
+            ]
+        },
+        "ROD5": {
+            "icao": "ROD5",
+            "name": "Rosewood Five",
+            "rwy": {
+                "3L": [],
+                "3R": [],
+                "4L": [],
+                "4R": [],
+                "9L": [],
+                "9R": [],
+                "21L": [],
+                "21R": [],
+                "22L": [],
+                "22R": [],
+                "27L": [],
+                "27R": []
+            },
+            "body": [["_DXO184010", "A50+"], "SCORR", "CHOOT"],
+            "exitPoints": {
+                "ROD": ["ROD"]
+            },
+            "draw": [
+                ["_DXO184010", "SCORR", "CHOOT", "ROD*"]
+            ]
+        },
+        "STCLR1": {
+            "icao": "STCLR1",
+            "name": "St. Clair One",
+            "rwy": {
+                "3L": [],
+                "3R": [],
+                "4L": [],
+                "4R": [],
+                "9L": [],
+                "9R": [],
+                "21L": [],
+                "21R": [],
+                "22L": [],
+                "22R": [],
+                "27L": [],
+                "27R": []
+            },
+            "body": [],
+            "exitPoints": {
+                "MAARS": [["_DXO106010", "A50+"], "MAARS"],
+                "PISTN": [["_DXO020010", "A50+"], "PISTN"],
+                "SCORR": [["_DXO184010", "A50+"], "SCORR"]
+            },
+            "draw": [
+                ["_DXO020010", "PISTN*"],
+                ["MAARS*"],
+                ["SCORR*"]
+            ]
+        }
+    },
+    "stars": {
+        "GEMNI4": {
+            "icao": "GEMNI4",
+            "name": "Gemini Four",
+            "entryPoints": {
+                "BKW": ["BKW", "TARCI", "VLADY", "JAMOX", "OTMAN", "BLISS", "BOBCT"],
+                "MOL": ["MOL", "BIGAL", "TARCI", "VLADY", "JAMOX", "OTMAN", "BLISS", "BOBCT"],
+                "SINDE": ["SINDE", "GEFFS", "VLADY", "JAMOX", "OTMAN", "BLISS", "BOBCT"]
```

Comment:
We can put `VLADY` and beyond into the body since they are all shared amongst all entries.